### ISSUE TITLE
Add Asset Management Documentation

### ISSUE THREAD

**Issue by robneu** [FlagshipWP/compass#53]
It's not 100% clear for people new to Grunt where assets such as images, fonts, and other static files should be added within the theme. We need to flesh out the Wiki for this a bit.

**Comment by franz-josef-kaiser** [comment]
Suggestion: Use a custom `config.json` and explain _that_ instead. Details in [this talk](https://docs.google.com/presentation/d/1xrQqBRf4dgbK6jH9EHBiNNw7MkEwPd6bsrQrcH7mHl4/edit#slide=id.g7b3e8dedb_0238) from WordCamp Praha.

**Comment by robneu** [comment]
Interesting. So, would the grunt config read that file for path definitions and whatnot rather than having them declared directly in the grunt config as we do right now? https://github.com/FlagshipWP/flagship-wp-theme-config/blob/develop/index.js

**Comment by franz-josef-kaiser** [comment]
Yes. And that would make it extremely portable and flexible. You just change the config, never the definition itself. Works even better with one task per file.
